src/main/flight/imu.c

```diff
@@ -449,7 +449,9 @@ static void imuCalculateEstimatedAttitude(timeUs_t currentTimeUs)
     float gyroAverage[XYZ_AXIS_COUNT];
     gyroGetAccumulationAverage(gyroAverage);
     float accAverage[XYZ_AXIS_COUNT];
-    accGetAccumulationAverage(accAverage);
+    if (!accGetAccumulationAverage(accAverage)) {
+        useAcc = false;
+    }
```

Comment:
@ledvinap , this should solve the acc bias problem when `accumulatedMeasurementCount == 0`.

Reply:
Yes, it should do. But is it worth the code complexity? Returning last sample/average should perform about the same (+see comment about LPF filtering)

Reply:
I'll merge this in as it is, and then add a PR to make this change.